en/user/articles-dxp/120-search/13-result-rankings.markdown

```diff
@@ -93,13 +92,10 @@ To create a new Result Rankings Alias:
     want to intercept (it can be a phrase, instead of just one word; and don't
     worry, you can add more later) in the _Search Query_ field. 
 
-    To limit the rankings intervention to just one index's results, enter the
-    index name in the Index Name field.
-
     Click _Customize Results_.
 
 A search query is executed. The results are displayed and the tools for pinning,
-hiding, re-ordering, and adding results are made available. First, consider
+hiding, and adding results are made available. Re-ordering First, consider
```

Comment:
I don't get this change, @rbohl .

Reply:
It was the beginning of a very promising sentence, whose promise was never fulfilled.